From 6151ba98ddf3a21ce09519678e85f59d3652836a Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 2 Nov 2020 15:12:57 +0000
Subject: [PATCH 12/31] client: cure musl header inclusion warning

In file included from ECNotifyMaster.cpp:18:
/usr/include/sys/signal.h:1:2: warning: #warning redirecting incorrect
"#include <sys/signal.h>" to "<signal.h>" [-Wcpp]
---
 provider/client/ECNotifyMaster.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/provider/client/ECNotifyMaster.cpp b/provider/client/ECNotifyMaster.cpp
index fb3354c23..f63a3a9c3 100644
--- a/provider/client/ECNotifyMaster.cpp
+++ b/provider/client/ECNotifyMaster.cpp
@@ -4,6 +4,7 @@
  */
 #include <kopano/platform.h>
 #include <algorithm>
+#include <csignal>
 #include <cstring>
 #include <mutex>
 #include <kopano/memory.hpp>
@@ -15,7 +16,6 @@
 #include <kopano/stringutil.h>
 #include "SOAPUtils.h"
 #include "WSTransport.h"
-#include <sys/signal.h>
 #include <sys/types.h>
 
 #define CALL_MEMBER_FN(object,ptrToMember) ((object).*(ptrToMember))
-- 
2.29.2

